starboard/build/config/win/BUILD.gn

```diff
@@ -41,6 +41,11 @@ config("common") {
     "UNICODE",
   ]
 
+  # Allows compilation against VS 2022.
+  if (is_win) {
+    configs = [ "//build/config/win:visual_studio_version_compat" ]
```

Comment:
You don't need `if (is_win)` (this is all windows-specific), and use `configs+=` instead of `configs=`

Reply:
Sorry, misled you here, it should not be `configs+=` because `configs` is only set conditionally. instead it should be `configs=` at the top of this config, then `configs+=` for adding the `no_pdb` config

Reply:
Ack, retrying the tests. I was not finished debugging this because it passed locally and I think it's because I have :no_pdbs being imported so the += operation worked for me. I think the github tests should pass now.